src/pages/ProAmmPool/PositionListItem.tsx

```diff
@@ -166,63 +169,41 @@ function PositionListItem({
 
   const { farms } = useElasticFarms()
 
-  // let farmAddress = ''
-  // let pid = ''
+  let farmAddress = ''
+  let pid = ''
```

Comment:
why

Reply:
?

Reply:
Why would you enable this again? I saw it previously disabled.